ui/dashboards/src/components/EmptyDashboard/EmptyDashboard.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Typography, Box } from '@mui/material';
+import { AddPanelButton } from '../AddPanelButton';
+import { EditVariablesButton } from '../Variables';
+
+export interface EmptyDashboardProps {
+  /**
+   * The title for the empty state. This should be relatively short text.
+   * If not specified, the default title will be used.
+   */
+  title?: string;
+
+  /**
+   * Descriptive text for the empty state. This can be longer text that gives the
+   * user additional information.
+   * If not specified, the default description will be used.
+   */
+  description?: string;
+
+  /**
+   * Additional text that will be displayed at the bottom of the empty state.
+   * If not specified, no additional text is shown.
+   */
+  additionalText?: string;
+
+  /**
+   * Components that are placed below the title and description that include
+   * actions for the user to take (e.g. buttons or links). If not specified,
+   * the default "add panel" and "add variable" buttons will be displayed. Set
+   * to `false` to disable the default buttons.
+   */
+  actions?: boolean | React.ReactNode;
+}
+
+const DEFAULT_TITLE = "Let's get started";
+const DEFAULT_DESCRIPTION = 'We currently support time series charts, gauge charts, stat charts and more!';
+const DEFAULT_ACTIONS = (
+  <>
+    <AddPanelButton variant="secondary" labelType="long" fullWidth={true} />
+    <EditVariablesButton variant="secondary" labelType="long" fullWidth={true} />
+  </>
+);
+
+// Constants from specifics in designs to make the default messaging look good.
+const EMPTY_CONTAINER_WIDTH = '450px';
```

Comment:
I would just call CONTAINER_WIDTH, since it's the container for the whole thing (everything in here relates to the EMPTY state, so doesn't necessarily need the prefix / it made me wonder for a second if this was an empty container)